Gopkg.toml

```diff
@@ -1,66 +1,62 @@
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
 required = [
-  "k8s.io/code-generator/cmd/client-gen",
+  "k8s.io/code-generator/cmd/defaulter-gen",
   "k8s.io/code-generator/cmd/deepcopy-gen",
-  "k8s.io/code-generator/cmd/informer-gen",
+  "k8s.io/code-generator/cmd/conversion-gen",
+  "k8s.io/code-generator/cmd/client-gen",
   "k8s.io/code-generator/cmd/lister-gen",
+  "k8s.io/code-generator/cmd/informer-gen",
+  "k8s.io/code-generator/cmd/openapi-gen",
+  "k8s.io/gengo/args",
 ]
 
-[[constraint]]
+[[override]]
+  name = "k8s.io/code-generator"
+  # revision for tag "kubernetes-1.12.3"
+  revision = "3dcf91f64f638563e5106f21f50c31fa361c918d"
+
+[[override]]
   name = "k8s.io/api"
-  branch = "release-1.9"
+  # revision for tag "kubernetes-1.12.3"
+  revision = "b503174bad5991eb66f18247f52e41c3258f6348"
```

Comment:
IMO.. it will be better if we use proper release tag instead of commit... but that's can be done later with subsequent PR's .